robyn/__init__.py

```diff
@@ -47,48 +48,52 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
             setup_reloader(self.directory_path, self.file_path)
             exit(0)
 
-        self.router = Router()
+        self.router = Router() 
         self.middleware_router = MiddlewareRouter()
         self.web_socket_router = WebSocketRouter()
         self.request_headers: List[Header] = []  # This needs a better type
         self.response_headers: List[Header] = []  # This needs a better type
         self.directories: List[Directory] = []
         self.event_handlers = {}
         self.exception_handler: Optional[Callable] = None
-        self.authentication_handler: Optional[AuthenticationHandler] = None
 
     def _add_route(
-        self,
-        route_type: HttpMethod,
-        endpoint: str,
-        handler: Callable,
-        is_const: bool = False,
-        auth_required: bool = False,
+        self, route_type: HttpMethod, endpoint: str, handler: Callable, is_const=False
     ):
         """
-        This is base handler for all the route decorators
+        This is base handler for all the decorators
 
-        :param route_type str: route type between GET/POST/PUT/DELETE/PATCH/HEAD/OPTIONS/TRACE
+        :param route_type str: route type between GET/POST/PUT/DELETE/PATCH
         :param endpoint str: endpoint for the route added
         :param handler function: represents the sync or async function passed as a handler for the route
-        :param is_const bool: represents if the handler is a const function or not
-        :param auth_required bool: represents if the route needs authentication or not
```

Comment:
And this?

Reply:
Our fork was outdated and didn't have any of the authentication changes + comments yet. Will fix that by the next commit.